src/qml/README.md

```diff
@@ -71,3 +71,9 @@ you must configure with the following option:
 ```
 ./configure --with-qml
 ```
+### Run
+
+To run the GUI QML application execute the file:
+```
+/src/qt/bitcoin-qt
```

Comment:
Now it refers to an absolute path which is pretty questionable.

Usually, it's preferred to use paths that are relative to the repo's directory:
```
./src/qt/bitcoin-qt
```
or
```
src/qt/bitcoin-qt
```

Reply:
The reason I chose not to use `./src/qt/bitcoin-qt` is because technically you could run the file from the src folder or also the qt folder directly by using `./bitcoin-qt`. However I can update it to whichever you prefer, for now I'll remove the initial forward slash to match your second option.

Reply:
Technically, I doubt if someone will place a repository into the system root `/` directory. That is why an absolute path like `/src/qt/bitcoin-qt` is meaningless in sane scenarios.

Btw, `./src/qt/bitcoin-qt` and `src/qt/bitcoin-qt` are equivalent, no?

> ... technically you could run the file from the src folder or also the qt folder directly by using `./bitcoin-qt`.

Yes. If your goal is to document the easiest way that is one I use everyday dozens of times:
```
./autogen.sh
./configure
make
./src/qt/bitcoin-qt
```

In Bash when one starts a command with `./` auto-completion works nice for executables :)

I believe that readers of this file will understand that this way to run Bitcoin Core is on testing purpose only.

Reply:
Ok sounds good, I think you have a good point which is that we want to document the most common and easiest way to run the file. I'll make the final change and then squash my commits again. Thanks!